packages/attestation-service/src/sms/twilio.ts

```diff
@@ -2,81 +2,34 @@ import bodyParser from 'body-parser'
 import Logger from 'bunyan'
 import express from 'express'
 import twilio, { Twilio } from 'twilio'
-import { fetchEnv, fetchEnvOrDefault } from '../env'
-import { AttestationModel, AttestationStatus } from '../models/attestation'
+import { fetchEnv } from '../env'
+import { AttestationStatus, SmsFields } from '../models/attestation'
 import { readUnsupportedRegionsFromEnv, SmsProvider, SmsProviderType } from './base'
 import { receivedDeliveryReport } from './index'
 
 export class TwilioSmsProvider extends SmsProvider {
```

Comment:
Nit: can be made `abstract`

Reply:
oop good callout! 🙏  (overlooked that I could remove the `fromEnv()` earlier and got a bit tripped up)